src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -470,6 +696,21 @@ static Log()
                    EventIds.OnCircuitClosed,
                    "Closing circuit with id {CircuitId}.");
 
+                _circuitHandlerFailed = LoggerMessage.Define<Type, string, string>(
+                    LogLevel.Error,
+                    EventIds.CircuitHandlerFailed,
+                    "Unhandled error invoking circuit handler type {handlerType}.{handlerMethod}: {Message}");
+
+                 _circuitUnhandledException = LoggerMessage.Define<string>(
+                    LogLevel.Error,
+                    EventIds.CircuitUnhandledException,
+                    "Unhandled exception in circuit {CircuitId}");
```

Comment:
```suggestion
                    "Unhandled exception in circuit {CircuitId}.");
```

Reply:
I'm rejecting all of these. 

Ending log messages with a period is not in our coding guidelines. Some areas of the product use a period consistently, and some omit it consistently. I'm following the existing style of the log messages in this file.

If you're deeply passionate about this, start a discussion with the whole team 😆

Reply:
Fine, I'll see what I can do.